content/cn/docs/admin/kubelet-authentication-authorization.md

```diff
@@ -51,7 +51,7 @@ To subdivide access to the kubelet API, delegate authorization to the API server
 * start the kubelet with the `--authorization-mode=Webhook` and the `--kubeconfig` flags
 * the kubelet calls the `SubjectAccessReview` API on the configured API server to determine whether each request is authorized
 
-The kubelet authorizes API requests using the same [request attributes](/docs/admin/authorization/#request-attributes) approach as the apiserver.
+The kubelet authorizes API requests using the same [request attributes](/docs/reference/access-authn-authz/authorization/) approach as the apiserver.
 
```

Comment:
The link target should be /docs/reference/access-authn-authz/authorization/#review-your-request-attributes

Reply:
Thought so, but wasn't 100% sure on that. I'll correct both.